### DISCUSSION TITLE
Introducing multiple measures to correlation-based SSM

### DISCUSSION THREAD

**Discussion by maditafruehauf** [jmgirard/circumplex#19] [Q&A]
Dear all, 
I am a PhD-candidate of the Freie Universität Berlin. I just started working with the circumplex package for analyzing circular data and I very much enjoy working with the package. 
I am currently performing  correlation-based SSM with interpersonal teacher behavior as circumplex measure and students emotions (anxiety) as well as students performance (via grades) as correlates. 
I did the following two analysis: First, I only introduces the anxiety variable. In the second analysis, I additionally added the students' grades. Now I did the observation that the SSM-parameters of the anxiety scales changed after adding grades as the second measure to the analysis (Contrast analysis revealed that the two measures significantly differed in their Y-value as well as in their displacement). 
I was wondering what the reason for the change is and how to interpret this change in SSM-parameters of the anxiety measure BEFORE and AFTER adding the grades to the analysis. In analogy to linear regressions with covariates, can I interpret the second model as "correlation of students' anxiety with interpersonal behavior while controlling for students' grades"? 
I would be happy if you could help the understand the underlying algorithm better in order to reliabily interpret this observation. 
Many thanks in advance!
Best 
Madita

**Comment by jmgirard** [comment]
Adding more than one measure should not change the SSM parameters because it is based on zero-order correlations not partial correlations. For instance, look at the NARPD estimates below when used alone vs. with ASPD; they are the same with only minor differences in the confidence intervals due to the randomness in bootstrapping (and we could get rid of that randomness if we wanted by setting a seed).

``` r
library(circumplex)
data("jz2017")
results1 <- ssm_analyze(
  .data = jz2017, 
  scales = c(PA, BC, DE, FG, HI, JK, LM, NO), 
  angles = octants(),
  measures = NARPD
)
results1
#> Call:
#> ssm_analyze(.data = jz2017, scales = c(PA, BC, DE, FG, HI, JK, 
#>     LM, NO), angles = octants(), measures = NARPD)
#> 
#> Profile [NARPD]:
#>                Estimate   Lower CI   Upper CI
#> Elevation         0.202      0.170      0.236
#> X-Value          -0.062     -0.096     -0.029
#> Y-Value           0.179      0.146      0.214
#> Amplitude         0.189      0.156      0.226
#> Displacement    108.967     99.201    118.857
#> Model Fit         0.957
results2 <- ssm_analyze(
  .data = jz2017, 
  scales = c(PA, BC, DE, FG, HI, JK, LM, NO), 
  angles = octants(),
  measures = c(NARPD, ASPD)
)
results2
#> Call:
#> ssm_analyze(.data = jz2017, scales = c(PA, BC, DE, FG, HI, JK, 
#>     LM, NO), angles = octants(), measures = c(NARPD, ASPD))
#> 
#> Profile [NARPD]:
#>                Estimate   Lower CI   Upper CI
#> Elevation         0.202      0.170      0.236
#> X-Value          -0.062     -0.094     -0.029
#> Y-Value           0.179      0.144      0.212
#> Amplitude         0.189      0.154      0.225
#> Displacement    108.967     99.042    118.156
#> Model Fit         0.957                      
#> 
#> Profile [ASPD]:
#>                Estimate   Lower CI   Upper CI
#> Elevation         0.124      0.089      0.158
#> X-Value          -0.099     -0.134     -0.064
#> Y-Value           0.203      0.169      0.238
#> Amplitude         0.226      0.191      0.262
#> Displacement    115.927    107.712    124.107
#> Model Fit         0.964
```

<sup>Created on 2023-02-13 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>
